interpreter/cling/test/Driver/Shebang.C

```diff
@@ -10,11 +10,9 @@
 // RUN: %cling %s 2>&1 | FileCheck %s
 
 //CHECK-NOT: {{.*error|warning|note:.*}}
-float shebang = 1.0
-//CHECK: (float) 1.0
+float shebang = 1.0 //CHECK: (float) 1
```

Comment:
Ouch. It would be really nice if the output could be `1.0f`. Can we get that somehow?